pkgs/tools/backup/luckybackup/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool
+, rsync, ssh
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  pname = "luckybackup";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/luckybackup/${version}/source/${pname}-${version}.tar.gz";
+    sha256 = "0nwjsk1j33pm8882jbj8h6nxn6n5ab9dxqpqkay65pfbhcjay0g8";
+  };
+
+  buildInputs = [ pkgconfig libtool rsync ssh ];
+  
+  prePatch = ''sed -i -e "s|/usr|$out|" luckybackup.pro'';
```

Comment:
Can someone help me here?

Is there a more "safe" way to do that patching? Something like a Nix function or script?